src/app/game-state/item-repo.service.ts

```diff
@@ -1167,6 +1169,29 @@ export class ItemRepoService {
         return this.homeService.autoBuyFurnitureUnlocked;
       }
     },
+    autoBuyerSettingsManual: {
+      id: 'autoBuySettingsManual',
+      name: "Manual of Customizaed Automation",
+      type: "manual",
+      description: "This manual teaches you to customize the order and behavior of auto-buying.",
+      value: 1000000000,
+      useLabel: "Read",
+      useDescription: "Permanently unlock auto-buying customization",
+      useConsumes: true,
+      use: () => {
+        if (!this.autoBuyerService){
+          this.autoBuyerService = this.injector.get(AutoBuyerService);
+        }
+        this.autoBuyerService.autoBuyerSettingsUnlocked = true;
+        this.logService.addLogMessage("The teachings of the manual sink deep into your soul. You'll be able to apply this knowledge in all future reincarnations.", "STANDARD", 'EVENT');
+      },
+      owned: () => {
+        if (!this.autoBuyerService){
+          this.autoBuyerService = this.injector.get(AutoBuyerService);
+        }
+        return this.autoBuyerService.autoBuyerSettingsUnlocked;
```

Comment:
Are we preserving this in the saved settings?

Reply:
It has a the get/set properties methods and it's added to gameState. So I think so!